src/components/registro/Registro.js

```diff
@@ -0,0 +1,332 @@
+          </Grid>
+
+          <Grid item xs={6}>
+            <TextValidator
+              validations={{ matchRegexp: emailRegex }}
+              id="emailUsuario"
+              label="Ingrese una correo electrónico"
+              name="email"
+              onChange={handleChange}
+              value={informacionDelUsuario.email}
+              validators={['required', 'isEmail']}
+              errorMessages={['Este campo es requerido', ERRORES.email]}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <EmailIcon />
+                  </InputAdornment>
+                ),
+              }}
+              style={{ minWidth: 250 }}
+            />
+          </Grid>
+
+          <Grid item xs={6} align="right">
+            <Typography variant="h6">Número de celular:</Typography>
+          </Grid>
+
+          <Grid item xs={6}>
+            <TextValidator
+              id="telefono"
+              label="Número de celular"
+              name="telefono"
+              onChange={handleChange}
+              value={informacionDelUsuario.telefono}
+              validators={['required']}
+              errorMessages={['Este campo es requerido', ERRORES.telefono]}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <PhoneEnabledIcon />
+                  </InputAdornment>
+                ),
+              }}
+              style={{ minWidth: 250 }}
+            />
+          </Grid>
+
+          <Grid item xs={6} align="right">
+            <Typography variant="h6">Contraseña:</Typography>
+          </Grid>
+
+          <Grid item xs={6}>
+            <TextValidator
+              id="contraseniaUsuario"
+              label="Ingrese una contraseña"
+              name="contrasenia"
+              type="password"
+              onChange={handleChange}
+              value={informacionDelUsuario.contrasenia}
+              validators={['required', 'minStringLength:6']}
+              errorMessages={['Este campo es requerido', ERRORES.contraseña]}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <LockIcon />
+                  </InputAdornment>
+                ),
+              }}
+              style={{ minWidth: 250 }}
+            />
+          </Grid>
+
+          <Grid item xs={6} align="right">
+            <Typography variant="h6">Confirmar contraseña:</Typography>
+          </Grid>
+
+          <Grid item xs={6}>
+            <TextValidator
+              id="confirmarContraseña"
+              label="Confirme la contraseña"
+              name="confirmarContraseña"
+              type="password"
+              onChange={handleChange}
+              value={informacionDelUsuario.confirmarContraseña}
+              validators={['required', 'minStringLength:6']}
```

Comment:
Si toman mi sugerencia de más arriba, acá habría que agregar `isPasswordMatch`. Lo del length se podría sacar porque ya lo define el campo contraseña:

```suggestion
              validators={['required', 'isPasswordMatch']}
```